pkg/kando/location.go

```diff
@@ -20,14 +20,24 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	"github.com/kanisterio/kanister/pkg/datamover"
 	"github.com/kanisterio/kanister/pkg/param"
 )
 
 const (
-	pathFlagName    = "path"
-	profileFlagName = "profile"
+	pathFlagName                         = "path"
+	profileFlagName                      = "profile"
+	repositoryServerFlagName             = "repository-server"
+	repositoryServerUserHostnameFlagName = "user-hostname"
```

Comment:
```suggestion
	repositoryServerUserHostnameFlagName = "repository-user-hostname"
```
?

Reply:
That's fine I guess. Since this flag is global, it should be better to differentiate.

Reply:
as per the discussion another option would be - `repository-server-user-hostname`

Reply:
Having `user-hostname` would suggest we are talking about a "user's hostname" rather than "user and hostname". I would recommend using `repository-server-user` as the flag name.
Basing my suggestion on the equivalent kopia command used to register users. It's `kopia server user add` where "user" is of format `username@hostname`

Reply:
@pavannd1 according to the kopia repository CR, we can have multiple users. And due to the design of it, for multiple users
- Username of all the users would be the same
- Only Hostname and Password would be different for each user

```
userAccess:
      userAccessSecretRef:
        name: repository-server-user-access
        namespace: kanister
      username: kanisteruser
```

Reply:
Moving on with `--client-hostname`